runtime/metrics_types.go

```diff
@@ -151,14 +149,13 @@ func NewMetric(name string, i interface{}) *Metric {
 		m.Measures["p95"] = p[2]
 		m.Measures["p99"] = p[3]
 		m.Measures["p999"] = p[4]
-		m.Measures["p9999"] = p[5]
 		m.Measures["m1"] = s.Rate1()
 		m.Measures["m5"] = s.Rate5()
 		m.Measures["m15"] = s.Rate15()
 		m.Measures["meanrate"] = s.RateMean()
 
 	default:
-		panic("unexpected metric type")
+		panic(fmt.Sprintf("unexpected metric type: %v", reflect.TypeOf(v)))
```

Comment:
Cool.

Reply:
I got the panic once, so I think its helpful :)